### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikolai-b**
Make the design image appear,
simplify the email-config.edn instructions.

I didn't understand the 
> point to it through the conf

part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +24,10 @@ Furthermore, Freecoin's first social wallet pilots are informed by the research
...
 - The conf can be found in project.clj
-- Add an email-conf.edn file and point to it through the conf in project.clj under profiles. It is needed for the system ti be able to send emails when needed (eg. when a user signs up). The file should be of the form:
-`{:email-server "" 
-  :email-user "" 
-  :email-pass "" 
+- Add an `email-conf.edn` file in the project root. It is needed for the system to be able to send emails when needed (e.g. when a user signs up). The file should be of the form:
+`{:email-server ""
+  :email-user ""
+  :email-pass ""
   :email-address ""}`
```
Comment (aspasia) [inline README.md]: Much better indeed, thanks

File: README.md [hunk 2]
```diff
@@ -9,8 +9,9 @@ Freecoin aims to be a framework for remuneration and authentication supporting m
 [![Code Climate](https://codeclimate.com/github/Commonfare-net/macao-social-wallet.png)](https://codeclimate.com/github/Commonfare-net/macao-social-wallet)
 
 ## Design
-https://freecoin.dyne.org/images/freecoin_logo.png
-[![Freecoin Cornucopia](
+
+<img src="https://freecoin.dyne.org/images/freecoin_logo.png" width="200" alt="Freecoin Cornucopia">
+
```
Comment (aspasia) [inline README.md]: Not sure if I understand why this is needed. "Make the design image appear": was it not showing for you?

Comment (aspasia) [reply README.md]: oh sorry I see now

**Comment by jaromil** [comment]
thanks nikolai!! and hi!! :^)

**Comment by nikolai-b** [comment]
Hey, Dan said I should be looking at what your building, :wave:

**Pull request merged**